.gitattributes

```diff
@@ -0,0 +1 @@
+*.ipynb linguist-vendored
```

Comment:
Nice, I don't know this feature ! May I suggest to add CSV and JSON as well ?

```suggestion
*.ipynb linguist-vendored
*.json linguist-vendored
*.csv linguist-vendored
```

Reply:
Good one! Adding it